src/routers/v3/RouteRouter.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import type Request from 'express';
+import AnalyticsUtils from '../../utils/AnalyticsUtils';
+import ApplicationRouter from '../../appdev/ApplicationRouter';
+import LogUtils from '../../utils/LogUtils';
+import RouteUtils from '../../utils/RouteUtils';
+
+class RouteRouter extends ApplicationRouter<Object> {
+  constructor() {
+    super(['POST']);
+  }
+
+  getPath(): string {
+    return '/route/';
+  }
+
+  async content(req: Request): Promise<Object> {
+    const {
+      destinationName,
+      end,
+      arriveBy,
+      originName,
+      start,
+      time,
+      uid,
+    } = req.body;
+
+    const isArriveBy = (arriveBy === '1' || arriveBy === true || arriveBy === 'true' || arriveBy === 'True');
```

Comment:
why are there so many possible inputs for `arriveBy`?

Reply:
that's a good question :) it's because people send in all types of random stuff to us so this is a precaution